network/manager.go

```diff
@@ -337,9 +337,9 @@ func (nm *networkManager) CreateEndpoint(cli apipaClient, networkID string, epIn
 	}
 
 	if nw.VlanId != 0 {
-		if epInfo.Data[VlanIDKey] == nil {
+		if epInfo[0].Data[VlanIDKey] == nil {
```

Comment:
is 1st entry will always be default nic? if so can you add comment here

Reply:
yup, added comment